prometheus/go_collector_latest.go

```diff
@@ -288,7 +288,7 @@ func NewGoCollector(opts ...func(o *internal.GoCollectorOptions)) Collector {
 }
 
 func attachOriginalName(desc, origName string) string {
-	return fmt.Sprintf("%s Sourced from %s", desc, origName)
```

Comment:
Can you rebase on release-1.20 branch @ArthurSens (I changed manually base branch in GH UI)